File: app/src/main/java/com/riders/thelab/data/local/model/weather/Coordinates.java
package com.riders.thelab.data.local.model.weather;

import com.google.gson.annotations.SerializedName;

import lombok.Getter;
import lombok.Setter;
import lombok.ToString;

@Getter
@Setter
@ToString
public class Coordinates {

    @SerializedName("lon")
    double longitude;

    @SerializedName("lat")
    double latitude;
}

File: app/src/main/java/com/riders/thelab/data/remote/dto/directions/Coordinates.kt
package com.riders.thelab.data.remote.dto.directions

import com.google.gson.annotations.SerializedName

class Coordinates(
    @SerializedName("lng")
    var longitude: Double = 0.0,

    @SerializedName("lat")
    var latitude: Double = 0.0,
)